packages/unlock-js/src/__tests__/integration/walletServiceIntegration.test.js

```diff
@@ -533,8 +533,33 @@ describe.each(UnlockVersionNumbers)('Unlock %s', (unlockVersion) => {
         let keysBefore
         let keyGrantees
         let transactionHash
+        let totalKeysBefore
+
         beforeAll(async () => {
           keyGrantees = [accounts[8], accounts[9]]
+
+          totalKeysBefore = await Promise.all(
+            keyGrantees.map((grantee) =>
+              web3Service.totalKeys(lockAddress, grantee)
+            )
+          )
+
+          // enable grant of multiple keys for same address for locks > v10
+          if (['v10'].indexOf(publicLockVersion) == -1) {
```

Comment:
This is not right. You should not add a version specific item here. All versions have a way to yield the number of keys per address.